### PR TITLE
WIP: Prevent registering with empty General Collateral

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#12130

*Description of changes:*
- If General Collateral has spaces - remove them to prevent registering with an empty description.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]

**Review by cameron-eyds — APPROVED** [review]

**Comment by dimak1** [comment]
@cameron-freshworks this was still a WIP branch.

**Comment by cameron-eyds** [comment]
> @cameron-freshworks this was still a WIP branch.

Shoot sorry. 
In the future please set to draft if it's not ready to go, at first glance it appeared to just be waiting on a second review.

**Pull request merged**
